lib/inspect_subpackages.c

```diff
@@ -9,6 +9,17 @@
 
 #include "rpminspect.h"
 
+/*
+ * Fotmat an RPM package name and an architecture into a space-separated string.
+ * A source package is denoted with "src" architecture.
+ */
+static void format_subpackage_name_arch(char **data, Header hdr) {
+    xasprintf(data, "%s %s",
+        headerGetString(hdr, RPMTAG_NAME),
+        headerIsSource(hdr) ? "src" : headerGetString(hdr, RPMTAG_ARCH)
```

Comment:
I had done this for other inspections too, I guess I just didn't use it here when I should have.  The function is get_rpm_header_arch() in rpm.c.  That can replace line 19.